.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: Run API tests
+on:
+  push:
+permissions:
+  id-token: write
+  contents: write
+  actions: write
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    environment: intg
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: coursier/cache-action@v6
+      - name: Configure AWS credentials from management account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.MANAGEMENT_ACCOUNT }}:role/GithubActionsRoleMgmt
+          aws-region: eu-west-2
+          role-session-name: APIDownloadDependencies
+      - name: Login to ECR
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.MANAGEMENT_ACCOUNT }}.dkr.ecr.eu-west-2.amazonaws.com
+      - name: Setup JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 8
```

Comment:
Are later versions of Java supported?

Reply:
This isn't even necessary, I've taken it out.